src/main/java/chess/domain/position/File.java

```diff
@@ -0,0 +1,52 @@
+package chess.domain.position;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+public enum File {
+
+    A(1),
+    B(2),
+    C(3),
+    D(4),
+    E(5),
+    F(6),
+    G(7),
+    H(8),
+    ;
+
+    private final int order;
+
+    File(int order) {
+        this.order = order;
+    }
+
+    public static int length() {
+        return values().length;
+    }
+
+    public int gap(File other) {
+        return Math.abs(order - other.order);
+    }
+
+    public List<File> findBetween(File other) {
+        if (this.order < other.order) {
+            return betweenAcs(other);
+        }
+        return betweenDesc(other);
+    }
+
+    private List<File> betweenAcs(File other) {
```

Comment:
Ascending을 표현한거라면 오타가 있네요!
```suggestion
    private List<File> betweenAsc(File other) {
```
기존에는 predicate를 활용했던걸로 아는데, 이렇게 메서드로 분리해두신 이유가 있나요??
그 외에도 두 값 중 min을 구해서 하나의 메서드로 처리해볼 수 있을 것 같아요